arm-insights/2015-04-01/swagger/insightsManagementClient.json

```diff
@@ -0,0 +1,2119 @@
+        "description": "Create or update the log profile.",
+        "parameters": [
+          {
+            "name": "name",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the log profile."
+          },
+          {
+            "name": "api-version",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "default": "2016-03-01",
+            "description": "The api-version to send to the server"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/LogProfile"
+            },
+            "description": "Parameters supplied to the operation."
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successful request to create or update a log profile"
+          },
+          "201": {
+            "description": "Created log profile"
+          },
+          "202": {
+            "description": "Accepted request to create or update log profile"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/microsoft.insights/logprofiles": {
+      "get": {
+        "tags": [
+          "LogProfiles"
+        ],
+        "operationId": "LogProfiles_List",
+        "description": "List the log profiles.",
+        "parameters": [
+          {
+            "name": "api-version",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "default": "2016-03-01",
+            "description": "The api-version to send to the server"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successful request to list log profiles",
+            "schema": {
+              "$ref": "#/definitions/LogProfileCollection"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": null
+        }
+      }
+    },
+    "/{resourceUri}/providers/microsoft.insights/diagnosticSettings/service": {
+      "delete": {
+        "tags": [
+          "ServiceDiagnosticSettings"
+        ],
+        "operationId": "ServiceDiagnosticSettings_Delete",
+        "description": "Deletes the diagnostic settings for the specified resource.",
+        "parameters": [
+          {
+            "name": "resourceUri",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The resource identifier of the configuration."
+          },
+          {
+            "name": "api-version",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "default": "2015-07-01",
```

Comment:
I tried separating the operations in files and generating them one by one, but I did not know how to generate a single IInsightsManagementClient containing all the references to the IXXXOperations coming from different spec files.
My generation process generated the IInsightsManagementClient over and over again for each spec files overwriting it every time.
Any suggestions on how to do that.